openapi.yaml

```diff
@@ -0,0 +1,675 @@
+# This is the Open API interface for Open Service Broker API. Every attempt will
+# be made to make the Open API version of OSB API accurately represent the
+# written specification. If the spec and this document conflict, the spec is
+# the authroity.
+
+openapi: "3.0.0"
+
+info:
+  title: Open Service Broker API
+  description: The Service Broker API defines an HTTP(S) interface between Platforms and Service Brokers.
+  license: 
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: 
+    name: Open Service Broker API
+    url: https://www.openservicebrokerapi.org/
+    email: open-service-broker-api@googlegroups.com
+  version: v2.13
+
+servers:
+  - url: http://example.com
+    description: Broker Endpoint
+  - url: http://localhost:3000
+    description: OSB Checker - MockOSB
+
+paths:
+  /v2/catalog:
+    get:
+      summary: get the service catalog
+      tags:
+        - Catalog
+      operationId: catalog.get
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+      responses:
+        '200':
+          description: catalog response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Catalog'
+
+  /v2/service_instances/{instance_id}:
+    put:
+      summary: provision a service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.provision
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id of instance to provision
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous operations supported
+          schema:
+            type: boolean
+      requestBody:
+        description: parameters for the requested service instance provision
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceInstanceProvisionRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceProvision'
```

Comment:
HTTP 200 responses shouldn't return the `operation` field

Reply:
The 200 could apply to the sync or async cases.

Reply:
Ah. I hadn't considered returning the id of a completed async operation (when a service instance has successfully been provisioned async).

Reply:
if I'm following this thread, upon an async provision a 202 is returned and it'll include the "operation". I don't think a 200 is ever returned for an async provision.  Or, if you're talking about the case of a resent provision and they asked for async, and its already done, then yes we'll get a 200 but w/o "operation" in the response body.